src/ResourceManager/StorageSync/Commands.StorageSync/ConfigSchema.json

```diff
@@ -0,0 +1,21 @@
+﻿{
```

Comment:
@afedyashov what is the purpose of this file?

Reply:
Contains configuration settings of our cmdlet. Currently they are embedded into the module (hardcoded). This might change in the future.